packages/cli/src/tools/envinfo.ts

```diff
@@ -16,6 +17,17 @@ async function getEnvironmentInfo(
 ): Promise<string | EnvironmentInfo> {
   const options = {json, showNotFound: true};
 
+  let packages = ['react', 'react-native', '@react-native-community/cli'];
+
+  const outOfTreePlatforms: {[key: string]: string} = {
+    win32: 'react-native-windows',
+  };
```

Comment:
maybe worth adding macos straight away, at least to justify using `os.platform` usage. cc @alloy

Reply:
Makes sense to me 👍